internal/parser/node_groups_test.go

```diff
@@ -0,0 +1,102 @@
+package parser_test
+
+import (
+	"errors"
+	"testing"
+
+	"github.com/stg-tud/bp2022_netlab/internal/experiment"
+	"github.com/stg-tud/bp2022_netlab/internal/movementpatterns"
+	"github.com/stg-tud/bp2022_netlab/internal/parser"
+	"github.com/stretchr/testify/assert"
+)
+
```

Comment:
please check in the file generally for the syntax of 
_, err := parser....
vs.
exp, err := parser....

Reply:
This is standard go syntax and in fact, it won't compile otherwise. In the cases where I used `_, err := ...`, only the `err` variable is relevant. But as it's the second return value of the function, there must be a first left hand operator – in that case the `_` operator